### ISSUE TITLE
Return To not working within recovery flow created by /admin/recovery/code

### ISSUE THREAD

**Issue by dan2kx** [ory/kratos#3050]
### Preflight checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions.
- [X] I agree to follow this project's [Code of Conduct](https://github.com/ory/kratos/blob/master/CODE_OF_CONDUCT.md).
- [X] I have read and am following this repository's [Contribution Guidelines](https://github.com/ory/kratos/blob/master/CONTRIBUTING.md).
- [ ] This issue affects my [Ory Network](https://www.ory.sh/) project.
- [X] I have joined the [Ory Community Slack](https://slack.ory.sh).
- [ ] I am signed up to the [Ory Security Patch Newsletter](https://ory.us10.list-manage.com/subscribe?u=ffb1a878e4ec6c0ed312a3480&id=f605a41b53).

### Describe the bug

When creating a recovery code for a user in the admin flow, the link generated directs the user to the self service recovery flow with a flow id, adding a return_to to the link does not take the user back to the the originating system.

I am using the admin recovery flow to create a user in a third party (hydra enabled) system and send them a custom recovery email with the code and would like them to return to my third party app when they are done. This did work when using the magic link strategy but not after switching to code based recovery.

The recovery flow with a return_to works just fine when not generated from the /admin/recovery/code api 

### Reproducing the bug

call /admin/recobery/code to generate a recovery_link and code, add a return_to to the generated link and follow the link to the application, enter the code and the new password, the user is left on the settings page and not returned to the return_to location

### Version

0.11.1

### On which operating system are you observing this issue?

Linux

### In which environment are you deploying?

Kubernetes with Helm

**Comment by aeneasr** [comment]
Could you please explain briefly where you added the return_to parameter?

**Comment by dan2kx** [comment]
I have discovered that passing the return_to into the client as a param works but is not documented and should maybe just be a body param? 

```node
 kratosAdmin
    .createRecoveryCodeForIdentity(
      {
        createRecoveryCodeForIdentityBody: {
          identity_id: identity,
          expires_in: expires,
        },
      },
      {
        params: {
          return_to: returnTo,
        },
      }
    )
```

**Comment by aeneasr** [comment]
True, that does work indeed, but is only working because of some internal funkiness (on accident). It wasn't done to work this way. But it still works :) I think this is a good suggestion, and we should definitely make it explicit. For the time being, one can probably use this "i guess it's a feature"
